rdr_service/app_util.py

```diff
@@ -182,6 +182,10 @@ def get_account_origin_id():
     return client_id
 
 
+def is_self_request():
+    return _is_self_request()
+
+
```

Comment:
Could it make more sense to rename the `_is_self_request` method to `is_self_request`?

Reply:
done